lib/src/subtree.c

```diff
@@ -940,27 +946,28 @@ static size_t ts_subtree__write_to_string(
     }
   }
 
-  if (is_visible) cursor += snprintf(*writer, limit, ")");
+  if (is_visible || (is_root && ts_subtree_child_count(self) > 0)) cursor += snprintf(*writer, limit, ")");
 
   return cursor - string;
 }
 
 char *ts_subtree_string(
   Subtree self,
+  TSSymbol alias_symbol,
```

Comment:
👍  It makes total sense that we'd need to pass this in. Could you add a Rust test for a calling `to_sexp()` on a node that is aliased?

Reply:
Yeah in python's suite a single `_newline` is aliased to `block`, that is tested in the rust test